static/app/views/replays/detail/focusArea.tsx

```diff
@@ -55,34 +86,15 @@ function FocusArea({}: Props) {
         />
       );
     case 'network': {
-      // Fake the span and Trace context
-      const nonMemorySpansEntry = {
-        type: EntryType.SPANS,
-        data: replay
-          .getRawSpans()
-          .filter(replay.isNotMemorySpan)
-          .map(({startTimestamp, endTimestamp, ...span}) => ({
-            ...span,
-            timestamp: endTimestamp,
-            start_timestamp: startTimestamp,
-            span_id: uuid4(), // TODO(replays): used as a React key
-            parent_span_id: 'replay_network_trace',
-          })),
-      };
-      const performanceEvent = {
-        ...event,
-        contexts: {
-          trace: {
-            type: 'trace',
-            op: 'Network',
-            description: 'WIP',
-            span_id: 'replay_network_trace',
-            status: 'ok',
-          },
-        },
-        entries: [nonMemorySpansEntry],
-      } as EventTransaction;
-      return <Spans organization={organization} event={performanceEvent} />;
+      return <Spans organization={organization} event={getPerformanceEvents()} />;
+    }
+    case 'network 2': {
+      return (
+        <NetworkList
+          event={getPerformanceEvents()}
+          startTimestamp={event?.startTimestamp}
+        />
+      );
     }
```

Comment:
<!-- GRAPHITE_METADATA {"trueLocation":{"startLine":78,"endLine":98,"prLink":"https://app.graphite.dev/github/pr/getsentry/sentry/36230"}} -->
<i class='graphite__hidden'>[Re: lines 78 to 98]<br/></i>

Nit: could add curlies to make the console case a block: `case 'console': { ... }`, and remove from the other two.
Curlies `{}` create a block, which creates scope for variables. So it's useful only when there are variables inside afaik.<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/getsentry/sentry/36230?utm_source=unchanged-line-comment">Graphite</a>.</span>